homeassistant/components/litejet/scene.py

```diff
@@ -33,6 +37,11 @@ def name(self):
         """Return the name of the scene."""
         return self._name
 
+    @property
+    def unique_id(self):
+        """Return a unique identifier for this scene."""
+        return str(self._index)
```

Comment:
If it makes sense to allow multiple entries, the unique id should include the config entry id or other unique identifier to ensure uniqueness between entries.